aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -0,0 +1,431 @@
+    },
+    "DeletionProtection": {
+      "type": "boolean",
+      "description": "A value that indicates whether the DB instance has deletion protection enabled. The database can't be deleted when deletion protection is enabled. By default, deletion protection is disabled."
+    },
+    "Domain": {
+      "type": "string",
+      "description": "The Active Directory directory ID to create the DB instance in. Currently, only MySQL, Microsoft SQL Server, Oracle, and PostgreSQL DB instances can be created in an Active Directory Domain."
+    },
+    "DomainIAMRoleName": {
+      "type": "string",
+      "description": "Specify the name of the IAM role to be used when making API calls to the Directory Service."
+    },
+    "EnableCloudwatchLogsExports": {
+      "type": "array",
+      "uniqueItems": true,
+      "insertionOrder": false,
+      "items": {
+        "type": "string"
+      },
+      "description": "The list of log types that need to be enabled for exporting to CloudWatch Logs. The values in the list depend on the DB engine being used."
+    },
+    "EnableIAMDatabaseAuthentication": {
+      "type": "boolean",
+      "description": "A value that indicates whether to enable mapping of AWS Identity and Access Management (IAM) accounts to database accounts. By default, mapping is disabled."
+    },
+    "EnablePerformanceInsights": {
+      "type": "boolean",
+      "description": "A value that indicates whether to enable Performance Insights for the DB instance."
+    },
+    "Engine": {
+      "type": "string",
+      "enum": [
+        "aurora",
+        "aurora-mysql",
+        "aurora-postgresql",
+        "mariadb",
+        "mysql",
+        "oracle-ee",
+        "oracle-se2",
+        "oracle-se1",
+        "oracle-se",
+        "postgres",
+        "postgresql-license",
+        "sqlserver-ee",
+        "sqlserver-se",
+        "sqlserver-ex",
+        "sqlserver-web"
+      ],
+      "description": "The name of the database engine that you want to use for this DB instance."
+    },
+    "EngineVersion": {
+      "type": "string",
+      "description": "The version number of the database engine to use."
+    },
+    "Iops": {
+      "type": "integer",
+      "minimum": 1000,
+      "description": "The number of I/O operations per second (IOPS) that the database provisions. The value must be equal to or greater than 1000."
+    },
+    "KmsKeyId": {
+      "type": "string",
+      "description": "The ARN of the AWS Key Management Service (AWS KMS) master key that's used to encrypt the DB instance."
+    },
+    "LicenseModel": {
+      "type": "string",
+      "enum": [
+        "bring-your-own-license",
+        "general-public-license",
+        "license-included",
+        "postgresql-license"
```

Comment:
Could you elaborate this LicenseModel or provide a doc? 
I didn’t find it under LicenseModel valid values here : https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html

Reply:
This came out of the blue while running contract tests:

`aws rds create-db-instance --db-name <db-name> --db-instance-identifier <db-instance-identifier> --allocated-storage 20 --engine postgres --db-instance-class db.t3.micro --master-username=<master-username> --master-user-password=<master-user-password>`

The output would contain:
```
"LicenseModel": "postgresql-license",
```

Strictly speaking, I have some doubts about this attribute: it is extremely fragile to adding new licenses and customers barely use it. Keeping in for the backwards compatibility.